models/enterprise_sonic/logging/deleted_example_01.txt

```diff
@@ -0,0 +1,29 @@
+# Using deleted
+#
+# Before state:
+# -------------
+#
+sonic# show logging servers
+--------------------------------------------------------------------------------
+HOST            PORT      SOURCE-INTERFACE    VRF            MESSGE-TYPE
+--------------------------------------------------------------------------------
+10.11.0.2       5         Ethernet24          -              event
+10.11.1.1       616       Ethernet8           -              log
+log1.dell.com   514       Ethernet28          -              log
+
+- name: Delete logging server configuration
+  sonic_logging:
+    config:
+      remote_servers:
+        - host: 10.11.1.2
```

Comment:
10.11.1.2 isn't a configured host. Apparently, this was intended to specify "10.11.0.2".

Reply:
Changed.